.semgrep/general.yaml

```diff
@@ -13,3 +13,39 @@ rules:
   languages: [ts]
   severity: ERROR
   fix: return $EXPR;
+
+- id: cyclomatic-complexity-whole-element
```

Comment:
Two new rules (actually one rule with two variations) to reduce cyclomatic complexity.